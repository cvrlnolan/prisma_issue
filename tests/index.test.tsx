/**
 * @jest-environment jsdom
 */

import React from "react";
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import Home from "../pages/index";

describe("Test Suite 1", () => {
  it("should render page correctly", () => {
    render(<Home />);
    screen.debug();
  });

  it("should match snapshot", () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
