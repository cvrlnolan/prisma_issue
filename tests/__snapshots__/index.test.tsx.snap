// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Suite 1 should match snapshot 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "minHeight": "100vh",
      "width": "100%",
    }
  }
>
  <p
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    Sample Prisma Issue
  </p>
</div>
`;
